src/main/java/com/eternalcode/combat/fight/effect/FightEffectController.java

```diff
@@ -70,13 +74,16 @@ public void onUntag(FightUntagEvent event) {
     }
 
     @EventHandler
-    public void onDeath(PlayerDeathEvent event) {
+    public void onDeath(FightUntagEvent event) {
```

Comment:
Listener above ^
Why are you listening to the same event twice? These listeners do the same thing.

Reply:
should I have one EventHandler for all cases of UnTag? 
Or is it better to split them like it is?

Reply:
The method `effectService#restoreActiveEffects` already performs `effectService#clearStoredEffects`, so why do it again?
![image](https://github.com/EternalCodeTeam/EternalCombat/assets/81445555/bfe9e28c-3493-4a0c-b884-c23391d8fa03)
![image](https://github.com/EternalCodeTeam/EternalCombat/assets/81445555/03c1ae5b-a3da-4c77-89f7-74ef64b51d72)

Reply:
Okay that has been fixed.
The issue was onQuit the user logsout with death screen. We dont want to add effects to his account, we want them cleared. I was asking should i combine does two listeners together? Also resolved the issue with "doing  the same thing
"
![image](https://github.com/EternalCodeTeam/EternalCombat/assets/80779749/2eed1c4c-4d7b-4501-b308-7de067741271)